ParkingZoneApp/Models/Entities/ParkingSlot.cs

```diff
@@ -0,0 +1,29 @@
+﻿using ParkingZoneApp.Models.Enums;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace ParkingZoneApp.Models.Entities
+{
+    [Table("ParkingSlot")]
+    public class ParkingSlot
+    {
+        [Key]
+        [Required]
+        public Guid Id { get; set; }
+
+        [Required]
+        public int Number { get; set; }
+
+        [Required]
+        public SlotCategory Category { get; set; } = SlotCategory.Standard;
```

Comment:
```suggestion
        public SlotCategory Category { get; set; }
```

Reply:
hmm, when you add Create page there should be category options to choose, thats why providing default value here would be little bit incorrect

Reply:
I thought that standard category will be mostly used and it will be good if the client does not have to choose the category most of the time but can switch any time he wants